src/coffee/directives/search-box.coffee

```diff
@@ -19,6 +19,7 @@ angular.module('uiGmapgoogle-maps')
 'uiGmapSearchBoxParentModel', '$http', '$templateCache', '$compile',
   (GoogleMapApi, Logger, SearchBoxParentModel, $http, $templateCache, $compile) ->
     class SearchBox
+      require: '^ngModel'
```

Comment:
Shouldn't this be "ngModel"? Since it's a sibling of uiGmapSearchBox? Rather than a parent?